schema/spacewalk/upgrade/susemanager-schema-4.2.21-to-susemanager-schema-4.3.0/002-new-timezones.sql

```diff
@@ -0,0 +1,7 @@
+insert into rhnTimezone(id, olson_name, display_name) ( 
```

Comment:
This whole file is not needed - for Uyuni it's enough, if you keep the other one (under the "latest" directory: 4.3.0 -> 4.3.1 currently).

Reply:
got it, thanks!